install/kube-rbac-proxy-kafka.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="kube-rbac-proxy-kafka"]
+= Configuring kube-rbac-proxy for Kafka
```

Comment:
```suggestion
= Configuring kube-rbac-proxy for Knative for Apache Kafka
```

Reply:
Same suggestion: "Knative Kafka".

Reply:
Resolved in favor of "Knative for Apache Kafka".